src/cpp/rtps/messages/MessageReceiver.cpp

```diff
@@ -1056,6 +1056,13 @@ bool MessageReceiver::proc_Submsg_Heartbeat(
     SequenceNumber_t lastSN;
     CDRMessage::readSequenceNumber(msg, &firstSN);
     CDRMessage::readSequenceNumber(msg, &lastSN);
+
+    SequenceNumber_t zeroSN;
+    if (fisrtSN <= zeroSN)
```

Comment:
This is not building
```suggestion
    if (firstSN <= zeroSN)
```